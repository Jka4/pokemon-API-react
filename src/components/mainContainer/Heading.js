import React from 'react';

const Heading = () => {
	return (
		<h2 className='titles' >
			<ul className='titlesList' >
				<li>photo</li>
				<li>name</li>
				<li>order</li>
				<li>exp</li>
			</ul>
		</h2>
	);
};

export default Heading