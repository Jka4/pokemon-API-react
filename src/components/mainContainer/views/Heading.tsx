import React from 'react';
import styled from 'styled-components';

const Heading: React.FC = () => {
  return (
    <Titles>
      <List>
        <Element>photo</Element>
        <Element>name</Element>
        <Element>order</Element>
        <Element>exp</Element>
      </List>
    </Titles>
  );
};

const Titles = styled.h2`
  width: 100%;
  max-width: 900px;
`;

const List = styled.ul`
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  grid-template-rows: 1fr;
  grid-column-gap: 20px;
  grid-row-gap: 0;
  padding-left: 0;
`;

const Element = styled.li`
  cursor: pointer;
  text-align: center;
  margin: 0;
  padding: 0;
  width: auto;
  list-style: none;
  text-transform: uppercase;
`;

export default Heading;
