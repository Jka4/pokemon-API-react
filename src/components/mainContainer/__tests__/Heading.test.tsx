import { mount } from 'enzyme';

import Heading from '../views/Heading';

describe('<Heading />', () => {
  it('renders correctly', () => {
    const wrapper = mount(<Heading />);
    expect(wrapper).toMatchSnapshot();
  });
});
