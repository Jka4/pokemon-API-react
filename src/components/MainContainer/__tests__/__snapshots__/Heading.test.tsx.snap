// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> renders correctly 1`] = `
<Heading>
  <styled.h2>
    <h2
      className="sc-bdfBwQ hqbntG"
    >
      <styled.ul>
        <ul
          className="sc-gsTCUz bkxspn"
        >
          <styled.li>
            <li
              className="sc-dlfnbm fdkJAr"
            >
              photo
            </li>
          </styled.li>
          <styled.li>
            <li
              className="sc-dlfnbm fdkJAr"
            >
              name
            </li>
          </styled.li>
          <styled.li>
            <li
              className="sc-dlfnbm fdkJAr"
            >
              order
            </li>
          </styled.li>
          <styled.li>
            <li
              className="sc-dlfnbm fdkJAr"
            >
              exp
            </li>
          </styled.li>
        </ul>
      </styled.ul>
    </h2>
  </styled.h2>
</Heading>
`;
